galata/test/jupyterlab/menus.test.ts

```diff
@@ -176,3 +176,180 @@ function reduceItem(
   }
   return null;
 }
+
+test.describe('Top menu keyboard navigation', () => {
```

Comment:
Should we tag this as an accessibility test?

```suggestion
test.describe('Top menu keyboard navigation @a11y', () => {
```